src/main/java/com/minecolonies/coremod/entity/citizen/citizenhandlers/CitizenChatHandler.java

```diff
@@ -47,78 +40,64 @@ public void notifyDeath(final DamageSource damageSource)
         if (citizen.getCitizenColonyHandler().getColony() != null && citizen.getCitizenData() != null)
```

Comment:
I wonder if we could get rid of this chat handler and make this a util method in the MsgUtils 
sendMessageFromCitizen(worker, ....)

Reply:
Switched to a builder-like setup, we could technically make this part of the MessageBuilder with a new `sendTo` overload?